cli/src/test/java/de/jplag/cli/MergingOptionsTest.java

```diff
@@ -0,0 +1,19 @@
+package de.jplag.cli;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import org.junit.jupiter.api.Test;
+
+import de.jplag.merging.MergingOptions;
+
+public class MergingOptionsTest extends CommandLineInterfaceTest {
+    @Test
```

Comment:
Add @Description("...") here. This is what is shown in your IDE. Can be a brief sentence describing the tested behavior (test if default values are used when cresting merging options).